packages/patternfly-4/react-core/src/components/DataList/DataList.md

```diff
@@ -16,6 +16,7 @@ import {
   DataListCell,
   DataListCheck,
   DataListAction,
+  DataListVisibility,
```

Comment:
I'm not 100% how you name classes in React, but I wonder if this should either be:
1. **DataListActionVisibility** if we intend to only give the user the option to change visibility for Data List actions, or
2. **Visibility** for consistency, as we name it that in the Table.

Reply:
Yes, you're right. Nevertheless, I will go with the former because I am using Data List Action modifiers.
Is that OK?

Reply:
Yes that sounds great @boaz0